tendermint/fundraising
# feat: initialize simulation functionality

## Description



Adds basic simulation functionality to the `fundraising` module.  This PR sets the stage for later simulation functionality to test the state transitions of the module.  The actual functionality is not added, as the functions in `x/fundraising/module.go` must be later completed.  

To test:
- run the new simulation `make` test targets
- run `starport chain simulate`

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [X] Targeted PR against correct branch
- [ ] Wrote unit and integration
- [ ] Added relevant `godoc` [comments](https://go.dev/blog/godoc).
